config.m4

```diff
@@ -434,6 +437,10 @@ if test "$MONGODB" != "no"; then
   if test "$PHP_LIBMONGOC" == "no"; then
     PHP_ADD_INCLUDE([$ext_srcdir/src/libmongoc/src/mongoc/])
     PHP_ADD_BUILD_DIR([$ext_builddir/src/libmongoc/src/mongoc/])
+    if test "x$with_zlib" = "xbundled"; then
+      PHP_ADD_INCLUDE([$ext_srcdir/src/libmongoc/src/zlib-1.2.11/])
+      PHP_ADD_BUILD_DIR([$ext_srcdir/src/libmongoc/src/zlib-1.2.11/])
```

Comment:
For the record, I'm not sure what `PHP_ADD_BUILD_DIR` does but I've added this line for consistency. Reading its definition in `acinclude.m4` (after running `phpize`), it looks like it just appends the the directory to `$BUILD_DIRS`, which is later used by PHP_GEN_BUILD_DIRS (called from our `configure.ac`) to ensure all of those directories exist (`mkdir` is called for them). The directory always exists, so this does seem redundant.

Reply:
These mkdirs make sense if you compile the extension outside of the source directory I believe. For example, you can do things like:

``
git clone ...
cd mongo-php-driver
phpize
mkdir build
cd build
../configure 
make
```

Reply:
If you're calling `phpize` from `mongo-php-driver/`, it's going to create `configure` in the project root. And then calling `../configure` from `mongo-php-driver/build` creates all of these directories under `build/` (essentially recreating much of the project source hierarchy) and the resulting object files will get dumped into those newly created directories. Is that correct?

Seems odd, but I suppose it makes sense.

Reply:
That's how I understand it. I've never seen the need for this, nor used this.